src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -841,16 +844,42 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
             // When doing invalid click
             await componentTestUtils.expectClickFailure('#chooseCoord_0_0', RulesFailure.MUST_CLICK_ON_EMPTY_SPACE());
 
-            // expect to see cancelMove reason as message
+            // Then the failure reason should be shown
             const expectedMessage: string = 'Perdu.';
             const currentMessage: string =
                 componentTestUtils.findElement('#currentMessage').nativeElement.innerHTML;
             expect(currentMessage).toBe(expectedMessage);
             const currentReason: string =
                 componentTestUtils.findElement('#currentReason').nativeElement.innerHTML;
             expect(currentReason).toBe(RulesFailure.MUST_CLICK_ON_EMPTY_SPACE());
-            // expect click to be still possible
-            expect(componentTestUtils.getComponent().canUserPlay('#chooseCoord_0_0').isSuccess()).toBeTrue();
+            // And then solution button should be shown too
+            componentTestUtils.expectElementToExist('#showSolutionButton');
+        }));
+        it('should show solution when asking for it', fakeAsync(async() => {
+            // Given a TutorialStep for clicks, for which the user clicked wrongly
+            const tutorial: TutorialStep[] = [
+                TutorialStep.forClick(
+                    'title',
+                    'Click on (0, 0)',
+                    QuartoState.getInitialState(),
+                    ['#chooseCoord_0_0'],
+                    'Bravo !',
+                    'Perdu.',
+                ),
+            ];
+            wrapper.startTutorial(tutorial);
+            await componentTestUtils.expectClickSuccess('#chooseCoord_1_1'); // click at the wrong place!
```

Comment:
The click is not the one expected by the tuto yet it is legal, hence "success"

C'est encore plus clair non ?